index.js

```diff
@@ -1,43 +1,97 @@
-let categoriesDiv = document.getElementById("categories-container");
+const body = document.querySelector("body");
+const categoriesDiv = document.getElementById("categories-container");
+
+/* Create category div with the default elements */
 
 let n = 0;
 
-window.addEventListener("load", addDefaultCategories);
+function addCategories() {
+  const newCategory = document.createElement("div");
+  newCategory.id = `category${n + 1}`;
+  newCategory.className = "categories";
 
-function addDefaultCategories() {
-  while (n < 5) {
-    let newCategory = document.createElement("div");
-    newCategory.id = `category${n + 1}`;
-    newCategory.className = "categories";
+  const categoryHeader = document.createElement("div");
+  categoryHeader.className = "category-header";
+  newCategory.appendChild(categoryHeader);
+
+  const listDiv = document.createElement("div");
+  listDiv.className = "list-div";
+  listDiv.id = `list${n + 1}`;
+  newCategory.appendChild(listDiv);
 
-    let categoryHeader = document.createElement("div");
-    categoryHeader.className = "category-header";
-    newCategory.appendChild(categoryHeader);
+  categoryTitle(categoryHeader);
 
-    let listDiv = document.createElement("div");
```

Comment:
maybe we can create reusable  function that creates a div with a class  name 
`const createDiv = (className) =>{
 let div =  document.createElement("div")
div.className = `${className}`;
}`
let listDiv = createDiv('list-div')

Reply:
good idea!